src/components/v2/common/ReactSelect.utils.tsx

```diff
@@ -33,19 +48,14 @@ export const styles = {
 }
 
 export function Option(props) {
-    const { selectOption, data, showTippy } = props
-    const style = { height: '16px', width: '16px', flex: '0 0 16px' }
-    const onClick = (e) => selectOption(data)
-    
+    const { selectProps, data, showTippy } = props
+    selectProps.styles.option = getCustomOptionSelectionStyle()
     const getOption = () => {
-        return <div className="flex left pl-12" style={{ background: props.isFocused ? 'var(--N100)' : 'transparent' }}>
-                {props.isSelected ? (
-                    <Check onClick={onClick} className="mr-8 icon-dim-16" style={style} />
-                ) : (
-                    <span onClick={onClick} className="mr-8" style={style} />
-                )}
+        return (
+            <div>
```

Comment:
do we have to keep this Option function after removing all this? If yes then can we remove the wrapping div?

Reply:
Using Tippy over component showing error. Hence, we are using div to wrap the component.

Reply:
Okay